backend/adapter_processor_v2/exceptions.py

```diff
@@ -51,6 +52,20 @@ class TestAdapterError(APIException):
     status_code = 500
     default_detail = "Error while testing adapter"
 
+    def __init__(
+        self,
+        sdk_err: SdkError,
+        detail: Optional[str] = None,
+        code: Optional[str] = None,
+        adapter_name: Optional[str] = None,
```

Comment:
I think , we could use kwargs for such details in future...

Reply:
Will incorporate it next time I make similar changes, thanks for the suggestion